content/en/docs/faq.md

```diff
@@ -5,6 +5,15 @@ type: docs
 weight: 8
 ---
 
+## What registries are compatible with Notary?
+
+The following registries are compatible with Notary for artifact signing and verification:
+
+- [Azure Container Registry](https://azure.microsoft.com/en-us/products/container-registry/)
+- [Docker Hub](https://hub.docker.com/)
```

Comment:
Docker Hub is not compatible at the moment.

Reply:
@toddysm Just to confirm, Docker has adopted `notation` per this: https://github.com/notaryproject/notaryproject.dev/pull/124/files#diff-a70d13bfec1091119e5df723c7213d64484d0a62069fe3c3f892a6418be523a2R19

but it is unavailable in dockerhub?

Reply:
Pushing Notation signatures to Docker Hub is failing as of right now. It used to work back in January.

Reply:
@toddysm Yes. I think we could remove DH at this moment.